dvc/command/render.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import logging
+
+from dvc.command.base import CmdBase, append_doc_link
+from dvc.utils.serialize._yaml import dumps_yaml, render_dvc_template
+
+logger = logging.getLogger(__name__)
+
+
+class CmdRender(CmdBase):
+    def run(self):
+        with open(self.args.path, encoding="utf-8") as fd:
+            text = fd.read()
+        dvc_dict, vars_dict = render_dvc_template(text)
+        vars_dict = {"vars": vars_dict}
+        vars_str = dumps_yaml(vars_dict)
+
+        if self.args.only_vars:
```

Comment:
Too many return points in one function. Can we make it more compact?

Reply:
Yeah definitely. This was tied Josh coding :-p